tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml

```diff
@@ -88,6 +116,8 @@ prepare: |
 
   #shellcheck disable=SC2148
   systemd-run --collect --unit fakedevicesvc fakedevicesvc localhost:11029
+  # stop fake device svc after we are done
+  tests.cleanup defer systemctl stop fakedevicesvc
```

Comment:
@sergiocazzolato you should compare this with https://github.com/snapcore/snapd/pull/10863, not sure which one we want to use

Reply:
about this, I just prefere to use the same in all these tests. I don't have any preference, both are ok, just updated in #10863 because all the similar tests are stoppng the fakedevicesvc in the restore.